docs/quickstart.rst

```diff
@@ -16,9 +16,15 @@ A Locust test is essentially just a Python program making requests to the system
             self.client.get("/hello")
             self.client.get("/world")
 
-This user will make an HTTP request to ``/hello``, then to ``/world``, and then repeat. For a full explanation and a more realistic example see :ref:`writing-a-locustfile`.
+This user will make an HTTP request to ``/hello``, then to ``/world``, and then repeat. By default, Locust repeats tasks indefinitely.
 
-Change ``/hello`` and ``/world`` to some actual paths on the website/service you want to test, put the code in a file named ``locustfile.py`` in your current directory and then run ``locust``:
+.. TODO suprisingly hard to find the source of this repetition, needs link / definition
+
+For a full explanation and a more realistic example see :ref:`writing-a-locustfile`.
+
+Change ``/hello`` and ``/world`` in the previous code block to two endpoints on the service you want to test, put the code in a file named ``locustfile.py`` in your current directory and then run ``locust``:
+
+.. TODO would it make sense to include a few lines of python you can run to serve two endpoints to test this with? Or do we assume that everyone who gets here can do that if they need to. `python3 -m http.server 9000` and empty `hello` and `world` files does the trick, not sure if you can make that a oneliner`
```

Comment:
I think having such an example makes a ton of sense! not sure what the best oneliner is...